box-ios-sdk-tests.podspec

```diff
@@ -0,0 +1,34 @@
+
+Pod::Spec.new do |s|
+
+  s.name         = "box-ios-sdk-tests"
+  s.version      = "1.0.0"
```

Comment:
1.0.5 to match the main podspec. We'll have to keep this updated I guess? Any way we can just make this point to the version in the main podspec?